packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -400,6 +406,13 @@ export const ChatScreen = () => {
                     ref={flatListRef}
                     onScrollToIndexFailed={handleScrollToIndexFailed}
                     refreshing={chat?.messagesStatus === Status.LOADING}
+                    maintainVisibleContentPosition={{
+                      minIndexForVisible: 0,
+                      autoscrollToTopThreshold:
+                        (chatContainerBottom.current -
+                          chatContainerTop.current) /
```

Comment:
fwiw on web I have this set to be only when exactly at the bottom (though it might have bugs)

Reply:
Noted. Couldn't get that behavior on mobile quickly so gonna leave it like this for now.